From 949658415f027adcd5dc0be0c165aa056952499a Mon Sep 17 00:00:00 2001
From: nitishat <nitisha.tomar@intel.com>
Date: Wed, 25 Mar 2020 01:21:49 +0530
Subject: [PATCH] USB formatting issue for Android in Container

When USB is plugged into the system it is shown as unsupported
and while trying to set it up by formatting the card the operation
never gets completed.

Tracked-On: OAM-90476

Change-Id: I1cd6a7bb517a32e55c44ed2d338fede683b1b9ce
Signed-off-by:Nitisha Tomar <nitisha.tomar@intel.com
---
 model/Disk.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/model/Disk.cpp b/model/Disk.cpp
index eb5f01f..b813ddb 100644
--- a/model/Disk.cpp
+++ b/model/Disk.cpp
@@ -122,7 +122,7 @@ Disk::Disk(const std::string& eventPath, dev_t device,
                 false), mJustPartitioned(false) {
     mId = StringPrintf("disk:%u,%u", major(device), minor(device));
     mEventPath = eventPath;
-    mSysPath = StringPrintf("/sys/%s", eventPath.c_str());
+    mSysPath = StringPrintf("/sys%s", eventPath.c_str());
     mDevPath = StringPrintf("/dev/block/vold/%s", mId.c_str());
     CreateDeviceNode(mDevPath, mDevice);
 }
-- 
2.25.1

